packages/cloudreverb/PKGBUILD

```diff
@@ -0,0 +1,73 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_name=CloudReverb
+pkgname=${_name,,}
+_plugin_uri='http://github.com/xunil-cloud/$_name'
+pkgver=0.2
+pkgrel=1
+pkgdesc='An algorithmic reverb plugin/app for creating huge, endless spaces and modulated echoes'
```

Comment:
Not a blocker, but in vim the `/` causes this to be marked faulty due to [PKGBUILD.vim](https://gitlab.archlinux.org/pacman/pacman-contrib/-/blob/master/src/vim/syntax/PKGBUILD.vim?ref_type=heads#L47) from pacman-contrib.